receiver/fluentforwardreceiver/config.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fluentforwardreceiver
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config defines configuration for the SignalFx receiver.
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
+
+	// Configures the receiver to use TLS.
+	// The default value is nil, which will cause the receiver to not use TLS.
+	// TLSCredentials *configtls.TLSSetting `mapstructure:"tls, omitempty"`
+
+	// The maximum number of connections to allow from upstream at a single
+	// time.
+	MaxConnections int `mapstructure:"max_connections,omitempty"`
```

Comment:
It is more just a self-protection mechanism for the collector in case something goes haywire sending.  Each connection is a separate goroutine with certain stack and heap requirements, but I'm not sure what the worst-case is.  I'm pretty sure the memory limiter in the collector won't really have much effect on this usage.  The OS will limit the connections to some extend I think due to port limits -- is it enough to rely on the OS limits?

Reply:
I took out the limiting.